src/Compilers/VisualBasic/Portable/Symbols/ReducedExtensionMethodSymbol.vb

```diff
@@ -947,6 +947,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Friend Overrides ReadOnly Property CallerArgumentExpressionParameterIndex As Integer
+            Get
+                Return m_CurriedFromParameter.CallerArgumentExpressionParameterIndex
```

Comment:
>Return m_CurriedFromParameter.CallerArgumentExpressionParameterIndex [](http://example.com/codeflow?start=16&length=68)

I do not think this is accurate. Does C# implements this property the same way? "Reduction" removes the first parameter. So, we probably should subtract 1 here. We should have tests for consuming the attribute on extension methods. Both for VB and C#. Even though at the moment C# doesn't use reduced methods during binding, they are exposed through SemanticModel and we can test API behavior on them.

Reply:
> So, we probably should subtract 1 here

Do you mean add?
If the first parameter is removed, then the second will have index 0, but we want to return 1. etc.
Or I'm misunderstanding?

Reply:
> Do you mean add?

No. The index is decreasing once we remove the first parameter. The second parameter becomes the first. Also the question what should we do about parameter that is removed and becomes the receiver, should its expression be used when the original index is 0.

Reply:
Indeed extension methods were broken. Fixed and added tests.